code/index.html

```diff
@@ -1,30 +1,82 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <!-- video header -->
+  <header>
+    <video id="video1" class="header-video" loop autoplay>
+      <source src="./videos/SampleVideo_1280x720_1mb.mp4" type="video/mp4">
+    </video>
+    <video id="video2" class="header-video" loop autoplay>
+      <source src="./videos/mediumVideo.mp4" type="video/mp4">
+    </video>
+    <div class="header-text">
+      <h1>Looking for a place to stay?</h1>
+      <h2>Try something different</h2>
+    </div>
+  </header>
+  <main>
     <!-- Signup form -->
+    <form id="signup-form" action="https://httpbin.org/anything" method="POST">
+      <div class="form-group">
+        <label for="name">Name</label>
+        <input id="name" type="text" name="Name">
+      </div>
+    <div class="form-group">
+        <label for="email">Email Address</label>
+        <input id="email" type="email" name="Email">
+      </div>
+      <div class="form-group">
+        <label for="password">Password</label>
+        <input id="password" type="password" name="Password">
+      </div>
+      <div class="form-group" id="daynightgroup">
+        <label for="dayornight">Day</label>
+        <input id="dayornight" type="radio" name="day or night">
+        <label for="dayornight">Night</label>
+        <input id="dayornight" type="radio" name="day or night">
+      </div>
+      <div class="form-group">
+        <label for="vegetarian">Vegetarian</label>
+        <input id="vegetarian" type="checkbox" name="vegetarian">
+      </div>
+      <div class="form-group">
+        <label for="submit">Book your spot!</label>
+        <input id="submit" type="submit" name="submit">
+      </div>
+    </form>
+
+    <!-- accordion with FAQ -->
 
     <section class="accordion">
       <!-- Accordion -->
-      <h1>FAQ</h1>
+      <h1 class="accordion-header">FAQ</h1>
+
+      <div class="question" id="section1">How much does it cost?</div>
+      <div class="answer">
+        <p>It all depends. If you are very nice to us we might give you an offer you can't refuse!</p>
+      </div>
```

Comment:
You've kept your accordion minimalistic, code-wise, which we appreciate. Looks very good 🌟